### PR TITLE
Add support for running troi after recommendations are run. 

### PR DESCRIPTION
**Pull request by mayhem**
This PR add the ability to run scripts as troi-bot once recommendations are complete. The only patch running right now is the one that sends a playlist to LB with a link to the model metadata.

The inner core of this code (which invokes troi) has been tested, but the rest... no idea how to do that.

### CODE REVIEW

File: listenbrainz/spark/troi_bot.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        Top level function called after spark CF recommendations have been completed.
+    """
+
+    if user in USERS_TO_PROCESS:
+        make_playlist_from_recommendations(user)
+        # Add others here
+
+def make_playlist_from_recommendations(user):
+
+    token = config.WHITELISTED_AUTH_TOKENS[0]
```
Comment (amCap1712) [inline listenbrainz/spark/troi_bot.py:20]: We should use `flask.current_app` to get config here.

Comment (mayhem) [reply listenbrainz/spark/troi_bot.py:20]: Is this code running in a flask context??

Comment (amCap1712) [reply listenbrainz/spark/troi_bot.py:20]: Yup, entire spark reader thus all handlers run in flask context (https://github.com/metabrainz/listenbrainz-server/blob/28b27f9a9bd3af933aa42d53848391ecebe4152f/listenbrainz/spark/spark_reader.py#L127)

Comment (mayhem) [reply listenbrainz/spark/troi_bot.py]: Updated.

**Comment by mayhem** [comment]
Thanks for catching the typos. Last week was really rough for me. :(

File: requirements.txt [hunk 2]
```diff
@@ -23,6 +23,7 @@ pika == 1.2.0
 git+https://github.com/metabrainz/brainzutils-python.git@v2.3.1
 spotipy == 2.17.1
 git+https://github.com/metabrainz/data-set-hoster.git@v-2021-02-10.0#egg=datasethoster
+git+https://github.com/metabrainz/troi-recommendation-playground.git#egg=troi
```
Comment (amCap1712) [inline requirements.txt:26]: Let's still pin it to some commit. lest we run the risk of breaking LB when working on troi.

Comment (mayhem) [reply requirements.txt]: Done.

Comment (amCap1712) [reply requirements.txt]: Thanks!

**Pull request merged**